pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1368,87 +1369,6 @@ func configAdapters(ctx *domainContext, config types.DomainConfig) error {
 	return nil
 }
 
-func createMountPointExecEnvFiles(containerPath string, mountpoints map[string]struct{}, execpath []string, workdir string, env []string, noOfDisks int) error {
```

Comment:
I don't even want to know how this got left in `domainmgr/domainmgr.go` and not moved to `domainmgr/containerd.go` ages ago (best guess? me :-) ). I am not complaining that it is getting moved correctly now.